shared/localization/format.js

```diff
@@ -12,6 +12,19 @@ import {getModuleDirectory} from '../esm-utils.js';
 import {isObjectOfUnknownValues, isObjectOrArrayOfUnknownValues} from '../type-verifiers.js';
 import {locales} from './locales.js';
 
+// From @formatjs/icu-messageformat-parser
+const TYPE = /** @type {const} */ ({
```

Comment:
It's not possible to import this? Can you leave a comment if this is the only way.